src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition/config.jelly

```diff
@@ -29,6 +29,9 @@ THE SOFTWARE.
     <f:entry field="scriptPath" title="${%Script Path}">
         <f:textbox default="Jenkinsfile"/>
     </f:entry>
+    <f:entry field="lightweight" title="${%Lightweight checkout}">
```

Comment:
I would think you would at most want an advanced option to *not* try lightweight. If you can do lightweight, why wouldn't you want to?